lib/ransack_ext.rb

```diff
@@ -9,4 +9,8 @@
   {'equals'=>'eq', 'greater_than'=>'gt', 'less_than'=>'lt'}.each do |old,current|
     config.add_predicate old, arel_predicate: current
   end
+
+  config.add_predicate 'lteqdate',
+  	arel_predicate: 'lt',
+  	formatter: -> (v) { v + 1.day }
```

Comment:
For  ``v + 1.day - 1.second`` predicate **lteq**, but for ``v + 1.day`` predicate **lt**
I think second variant better,